client/package.json

```diff
@@ -115,27 +110,26 @@
   },
   "scripts": {
     "vscode:prepublish": "tsc -p ./",
-    "compile": "tsc -watch -p ./",
+    "compile": "tsc -p ./",
+    "compile-watch": "tsc -watch -p ./",
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "node ./node_modules/vscode/bin/test"
   },
+
   "devDependencies": {
     "typescript": "^2.0.3",
     "vscode": "^1.0.0",
-    "vscode-languageclient": "^3.1.0",
     "vsce": "^1.18.0",
-    "mocha": "^2.3.3",
+
+    "@types/node": "^6.0.40",
```

Comment:
Ok.  because I couldn't put in comments (coz JSON) I used spacing and grouping to help make it look obvious.  gulp stuff was together.  typescript types were together.  I can change this back to what it was, or split into a separate commits.